<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateView</class>
 <widget class="QWidget" name="CreateView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Route</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>219</y>
     <width>361</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="bottomButtonLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="createButton">
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>169</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="viewNamehorizontalLayout">
    <item>
     <widget class="QLabel" name="viewNameLabel">
      <property name="text">
       <string>View Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="viewNameLineEdit">
      <property name="placeholderText">
       <string>View</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="viewPathhorizontalLayout">
    <item>
     <widget class="QLabel" name="viewPathLabel">
      <property name="text">
       <string>View Path</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="viewPathLineEdit">
      <property name="placeholderText">
       <string>path</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
